table-binary-unicode-properties.html

```diff
@@ -158,23 +158,38 @@
       <td><a href="https://unicode.org/reports/tr51/#Emoji_Properties">`Emoji`</a></td>
     </tr>
     <tr>
-      <td>`Emoji_Component`</td>
+      <ul>
```

Comment:
The markup is messed up here - these need to be inside of a `<td>`. (See the [preview](https://deploy-preview-1939--ecma262-snapshots.netlify.app/#table-binary-unicode-properties).)